test/Microsoft.Azure.WebJobs.Host.FunctionalTests/TestDoubles/MemoryBlobStore.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.WebJobs.Host.FunctionalTests.TestDoubles
     internal class MemoryBlobStore
     {
         private readonly ConcurrentDictionary<string, Container> _items = new ConcurrentDictionary<string, Container>();
-        private ServiceProperties _properties = new ServiceProperties();
+        private ServiceProperties _properties = new ServiceProperties(new LoggingProperties(), new MetricsProperties(), new MetricsProperties(), new CorsProperties());
```

Comment:
Yeah, I'm not sure either - I haven't looked at this test code before. Since its just test code, I'd say if all the tests are passing with what you have, that's good.